utils.py

```diff
@@ -265,20 +268,30 @@ def get_package_manager():
 
     # Check for package manager and associated packages
     if shutil.which('apt') is not None: # debian, ubuntu
-        config.PACKAGE_MANAGER_COMMAND = "apt install -y"
+        config.PACKAGE_MANAGER_COMMAND_INSTALL = "apt install -y"
+        config.PACKAGE_MANAGER_COMMAND_QUERY = "dpkg -l | grep -E ^"
```

Comment:
Needs testing

Reply:
The `dpkg` command would need to be more like `dpkg -l | grep '^.i  '` (two spaces after the "i"). Or, if we want to go pythonic, we could use the [Python APT library](https://apt-team.pages.debian.net/python-apt/library/index.html) like this, if we're checking if `pkg_name` is installed:
```python
try:
    import apt_pkg
except ModuleNotFoundError: # not debian/ubuntu
    pass
cache = apt_pkg.Cache()
installed_ver = cache[pkg_name].current_ver # None if not installed
```

Reply:
I'm going to stick with this method since it works for the other distros, but I've made a note in the code to your comment for future consideration.